devstack_vm/bin/run-all-tests.sh

```diff
@@ -37,7 +37,7 @@ fi
 $basedir/parallel-test-runner.sh $tests_file $tests_dir $log_file \
     $parallel_tests $max_attempts || true
 
-if [[ $project == "nova" ]]; then
```

Comment:
Remove this. It should (for all projects) test only if the file exists. If there is any isolated-tests.txt handle it.